TVUIComponent/lib/src/main/java/com/amazon/android/uamp/ui/PlaybackTrailerActivity.java

```diff
@@ -0,0 +1,2241 @@
+import com.amazon.android.contentbrowser.ContentBrowser;
+import com.amazon.android.contentbrowser.database.helpers.RecentDatabaseHelper;
+import com.amazon.android.contentbrowser.database.helpers.RecommendationDatabaseHelper;
+import com.amazon.android.contentbrowser.database.records.RecentRecord;
+import com.amazon.android.contentbrowser.helper.AnalyticsHelper;
+import com.amazon.android.model.content.Content;
+import com.amazon.android.module.ModuleManager;
+import com.amazon.android.recipe.Recipe;
+import com.amazon.android.tv.tenfoot.R;
+import com.amazon.android.uamp.DrmProvider;
+import com.amazon.android.uamp.UAMP;
+import com.amazon.android.uamp.constants.PreferencesConstants;
+import com.amazon.android.uamp.helper.CaptioningHelper;
+import com.amazon.android.uamp.mediaSession.MediaSessionController;
+import com.amazon.android.ui.fragments.ErrorDialogFragment;
+import com.amazon.android.utils.ErrorUtils;
+import com.amazon.android.utils.Helpers;
+import com.amazon.android.utils.Preferences;
+import com.amazon.mediaplayer.AMZNMediaPlayer;
+import com.amazon.mediaplayer.AMZNMediaPlayer.PlayerState;
+import com.amazon.mediaplayer.playback.text.Cue;
+import com.amazon.mediaplayer.tracks.MediaFormat;
+import com.amazon.mediaplayer.tracks.TrackType;
+import com.amazon.utils.DateAndTimeHelper;
+import com.google.android.exoplayer.text.CaptionStyleCompat;
+import com.google.android.exoplayer.text.SubtitleLayout;
+import com.zype.fire.api.IZypeApi;
+import com.zype.fire.api.Model.AdvertisingSchedule;
+import com.zype.fire.api.Model.ErrorBody;
+import com.zype.fire.api.Model.PlayerData;
+import com.zype.fire.api.Model.PlayerResponse;
+import com.zype.fire.api.Util.AdMacrosHelper;
+import com.zype.fire.api.Util.ErrorHelper;
+import com.zype.fire.api.ZypeApi;
+import com.zype.fire.api.ZypeSettings;
+import com.zype.fire.auth.ZypeAuthentication;
+
+import java.net.CookieHandler;
+import java.net.CookieManager;
+import java.net.CookiePolicy;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper;
+
+/**
+ * PlaybackOverlayActivity for content playback that loads PlaybackOverlayFragment
+ */
+public class PlaybackTrailerActivity extends Activity implements
+        PlaybackOverlayFragment.OnPlayPauseClickedListener, AMZNMediaPlayer
+        .OnStateChangeListener, AMZNMediaPlayer.OnErrorListener, AMZNMediaPlayer.OnInfoListener,
+        AudioManager.OnAudioFocusChangeListener, AMZNMediaPlayer.OnCuesListener,
+        ErrorDialogFragment.ErrorDialogFragmentListener {
+
+    private static final int TRANSPORT_CONTROLS_DELAY_PERIOD = 50;
+    private static final String TAG = PlaybackActivity.class.getSimpleName();
```

Comment:
I guess we should use `PlaybackTrailerActivity.class` here

Reply:
fixed